src/utils/mock.ts

```
@@ -0,0 +1,70 @@
+    let result = 0;
+    for (let i = 0; i < count; i++) {
+      result += value * i;
+    }
+    return result;
+  }
+
+  formatDate(date: Date | null | undefined): string {
+    if (!date) {
+      throw new Error('Date cannot be null or undefined');
+    }
+    if (!(date instanceof Date) || isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)